types/twitter-stream-channels/index.d.ts

```diff
@@ -48,12 +48,12 @@ declare module 'twitter-stream-channels' {
       /**
        * Closes the opened stream with Twitter
        * @method stop
-       * @param {StreamChannelsOptions} [options]
-       * @param {StreamChannelsOptions} [options.removeAllListeners=false] If true removes all the listeners set on the stream
+       * @param {Object} [options]
```

Comment:
Please use `object`, not `Object`, as discussed [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).